src/__tests__/proposals.int.test.ts

```diff
@@ -78,6 +78,103 @@ describe('/proposals', () => {
         );
     });
 
+    it('returns according to pagination parameters', async () => {
+      await db.query(`
+        INSERT INTO opportunities (
+          title,
+          created_at
+        )
+        VALUES
+          ( '🔥', '2525-01-02T00:00:01Z' )
+      `);
```

Comment:
Do we care about this timestamp? Can we use the existing insert script instead?

```suggestion
      await db.sql('opportunities.insertOne', { title: '🔥' });
```

Reply:
Oh hey I like that way better.